Sources/UIView+Sweet.swift

```diff
@@ -70,6 +70,15 @@
         class func instanceFromNib<T: UIView>(bundle: Bundle = .main) -> T {
             return UINib(nibName: String(describing: T.self), bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as! T
         }
+        
+        func pinToSuperview(top: CGFloat = 0.0, left: CGFloat = 0.0, bottom: CGFloat = 0.0, right: CGFloat = 0.0) {
```

Comment:
I think we already have this:  https://github.com/UseSweet/SweetUIKit/blob/a4472d892627df891de23d7f48c508a0ba157e3a/Sources/UIView%2BSweet.swift#L33